README.md

```diff
@@ -11,6 +11,24 @@ Mrs. Landingham will 18F employees about working in the federal government, how
 
 Please read the [contribution guidelines](CONTRIBUTING.md) before submitting a pull request.
 
+### Not sure what to work on?
+
+Check out the [Dolores Bot Waffle.io
+board](https://waffle.io/18F/dolores-landingham-bot) and pick out something in
+the "Up next" column. Make sure you move the feature you're working on the "In
+progress" so we all know who is working on what.
```

Comment:
`Make sure you move the feature you're working on the "In
progress" and assign it to yourself so we all know who is working on what.`